gobblin-metrics/src/main/java/gobblin/metrics/kafka/SchemaUtils.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.metrics.kafka;
+
+import java.util.List;
+
+import org.apache.avro.Schema;
+
+import com.google.common.base.Function;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+
+import javax.annotation.Nullable;
+
+
+/**
+ * Utilities for {@link org.apache.avro.Schema}.
+ */
+public class SchemaUtils {
+
+  /**
+   * Copies the input {@link org.apache.avro.Schema} but changes the schema name.
+   * @param schema {@link org.apache.avro.Schema} to copy.
+   * @param newName name for the copied {@link org.apache.avro.Schema}.
+   * @return A {@link org.apache.avro.Schema} that is a copy of schema, but has the name newName.
+   */
+  public static Schema switchName(Schema schema, String newName) {
```

Comment:
You can put this method in `AvroUtils`. There are other schema-related methods in that class.